mulighetsrommet-arena-adapter/src/main/kotlin/no/nav/mulighetsrommet/arena/adapter/plugins/DependencyInjection.kt

```diff
@@ -157,7 +141,17 @@ private fun services(services: ServiceConfig, tokenClient: MachineToMachineToken
             tokenClient.createMachineToMachineToken(services.arenaOrdsProxy.scope)
         }
     }
-    single { ArenaEventService(services.arenaEventService, get(), get()) }
+    single {
+        val processors = listOf(
+            TiltakEventProcessor(get(), get()),
```

Comment:
Er det noen grunn til at det er to av denne?

Reply:
Nei :grimacing:  Bra spotta :eyes: